README.md

```diff
@@ -52,9 +53,13 @@ Die Landeshauptstadt München beschafft so professionelle Softwarelösungen, ohn
 So können Leistungen jederzeit neu ausgeschrieben und bestmöglich am Markt vergeben werden.
 
 ### Client-Betriebssysteme und Office-Anwendungen
+<!-- Habe ich als Punkt übernommen - finde es aber schwierig hier etwas darüber zu schreiben, nachdem gerade Libre Office und Linux - mit nicht unerheblichem Aufwand - durch MS Office und Windows ersetzt wird. -->
 
```

Comment:
Ich weiß, uns war es aber sehr wichtig, im Koalitionsvertrag festzuhalten, dass auch das nicht in Stein gemeißelt ist. Persönlich möchte ich nicht zurück zu LiMux, sondern eher in Richtung virtualisierte Desktops gehen, wie sie z.B. das Land Schleswig-Holstein einsetzt, in Verbindung mit LibreOffice statt MS Office. Der Koalitionspartner ist erwartungsgemäß anderer Meinung.